### PR TITLE
V21 cherrypicks

### PR DESCRIPTION
**Pull request by ShimShtein**

### CODE REVIEW

**Comment by ShimShtein** [comment]
@laviro, we want to rebase and merge this one. The cherry-picks went smooth, without conflicts.

File: lib/foreman_rh_cloud/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module ForemanRhCloud
-  VERSION = '3.0.20'.freeze
+  VERSION = '3.0.21'.freeze
```
Comment (laviro) [inline lib/foreman_rh_cloud/version.rb:2]: should it be `3.0.21.1`?

**Review by laviro — APPROVED** [review]
Thanks @ShimShtein, tested and didn't find any issues

**Comment by ShimShtein** [comment]
Now against the correct branch and with correct workflow.
@laviro , ready for another round

**Comment by laviro** [comment]
Thanks @ShimShtein also for the workflow fix!

**Pull request merged**